lib/viewvc.py

```diff
@@ -91,6 +91,14 @@
 def cmp(a, b):
   return (a > b) - (a < b)
 
+class TextIOWrapper_noclose(io.TextIOWrapper):
+  """Custom TextIOWrapper class which don't close underlaid IO object when
```

Comment:
s/don't/doesn't/
s/underlaid/underlying/

+1 to commit with those tweaks.

Reply:
Thank you for the review. I commited it with those fixes.